app/src/main/res/drawable/pointer.xml

```diff
@@ -0,0 +1,10 @@
+<vector android:height="24dp" android:viewportHeight="72.0"
```

Comment:
nit: license.

fwiw, you can set up AS to automatically insert file headers (Settings -> Editor -> "File and Code Templates") but I've had bad experiences where it unsets itself (maybe on IDE updates?).